lmdeploy/serve/async_engine.py

```diff
@@ -286,15 +286,15 @@ async def get_generator(self, stop: bool, session_id: int):
         self.running_session_ids.add(session_id)
         return generator
 
-    def batch_infer(self,
-                    prompts: Union[List[str], str, List[Dict],
-                                   List[List[Dict]]],
-                    gen_config: Optional[Union[GenerationConfig,
-                                               List[GenerationConfig]]] = None,
-                    do_preprocess: bool = True,
-                    adapter_name: Optional[str] = None,
-                    use_tqdm: bool = False,
-                    **kwargs):
+    async def async_batch_infer(
```

Comment:
Is this API purely for multithread migration?

Reply:
Yes